packages/demo/src/components/NonModalView.tsx

```diff
@@ -1,24 +1,22 @@
-import { Paper, Stack } from '@mui/material'
-import { ModalToggle } from './ModalToggle'
-import { ValueMutator } from './ValueMutator'
-import { HeightChangeDemo } from './HeightChangeDemo'
-import { UpdaterFunctionDemo } from './UpdaterFunctionDemo'
-import { AssetSelector } from './AssetSelector'
-import { ContextRequester } from './ContextRequester'
-import { PluginComponent } from './FieldPluginDemo'
-import { LanguageView } from './LanguageView'
+import { Paper, Stack } from "@mui/material";
+import { ModalToggle } from "./ModalToggle";
+import { ValueMutator } from "./ValueMutator";
+import { HeightChangeDemo } from "./HeightChangeDemo";
+import { AssetSelector } from "./AssetSelector";
+import { ContextRequester } from "./ContextRequester";
+import { PluginComponent } from "./FieldPluginDemo";
+import { LanguageView } from "./LanguageView";
 
 export const NonModalView: PluginComponent = (props) => (
   <Paper>
     <Stack gap={6}>
       <ModalToggle {...props} />
       <ValueMutator {...props} />
-      <UpdaterFunctionDemo {...props} />
       <ModalToggle {...props} />
       <AssetSelector {...props} />
       <ContextRequester {...props} />
       <HeightChangeDemo {...props} />
       <LanguageView {...props} />
     </Stack>
   </Paper>
-)
+);
```

Comment:
Strange that this passed the checks from prettier in the GitHub workflow 🤔  

The strings above also use `""` instead of `''`.

Reply:
yeah I'll take a look at the github workflow. something must be missing. fixed here, anyway!